mono/utils/mono-context.h

```diff
@@ -33,6 +33,7 @@ typedef struct __darwin_xmm_reg MonoContextSimdReg;
 typedef struct _libc_xmmreg MonoContextSimdReg;
 #elif defined(HOST_WIN32)
 #define MONO_HAVE_SIMD_REG
+//#define MONO_HAVE_SIMD_REG_AVX
```

Comment:
is this intentionally commented out?

Reply:
I had copied the mono-context.h from runtimelab and that came with it and it wasn't my change. Reversed.